mevislab.github.io/package-lock.json

```diff
@@ -51,7 +51,7 @@
     "bootstrap": {
       "version": "5.2.2",
       "resolved": "https://registry.npmjs.org/bootstrap/-/bootstrap-5.2.2.tgz",
-      "integrity": "sha512-dEtzMTV71n6Fhmbg4fYJzQsw1N29hJKO1js5ackCgIpDcGid2ETMGC6zwSYw09v05Y+oRdQ9loC54zB1La3hHQ==",
+      "integrity": "sha512-tmhPET9B9qCl8dCofvHeiIhi49iBt0EehmIsziZib65k1erBW1rHhj2s/2JsuQh5Pq+xz2E9bEbzp9B7xHG+VA==",
```

Comment:
why is that changed, and why does it still work? 🤔

Reply:
I think it might have changed itself depending on something somewhere. I'm not completely sure but I guess as long as it works....